src/components/tx-flow/flows/EnableRecovery/EnableRecoveryFlowSettings.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  Divider,
+  CardActions,
+  Button,
+  Typography,
+  SvgIcon,
+  MenuItem,
+  TextField,
+  Collapse,
+  Checkbox,
+  FormControlLabel,
+} from '@mui/material'
+import ExpandLessIcon from '@mui/icons-material/ExpandLess'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { useForm, FormProvider, Controller } from 'react-hook-form'
+import { useState } from 'react'
+import type { TextFieldProps } from '@mui/material'
+import type { ReactElement } from 'react'
+
+import TxCard from '../../common/TxCard'
+import { EnableRecoveryFlowFields, RecoveryDelayPeriods, RecoveryExpirationPeriods } from '.'
+import AddressBookInput from '@/components/common/AddressBookInput'
+import CircleCheckIcon from '@/public/images/common/circle-check.svg'
+import { useDarkMode } from '@/hooks/useDarkMode'
+import type { EnableRecoveryFlowProps } from '.'
+
+import commonCss from '@/components/tx-flow/common/styles.module.css'
+import css from './styles.module.css'
+
+export function EnableRecoveryFlowSettings({
+  params,
+  onSubmit,
+}: {
+  params: EnableRecoveryFlowProps
+  onSubmit: (formData: EnableRecoveryFlowProps) => void
+}): ReactElement {
+  const [showAdvanced, setShowAdvanced] = useState(params[EnableRecoveryFlowFields.txExpiration] !== '0')
+  const [understandsRisk, setUnderstandsRisk] = useState(false)
+  const isDarkMode = useDarkMode()
+
+  const formMethods = useForm<EnableRecoveryFlowProps>({
+    defaultValues: params,
+    mode: 'onChange',
+  })
+
+  const emailAddress = formMethods.watch(EnableRecoveryFlowFields.emailAddress)
+
+  const onShowAdvanced = () => setShowAdvanced((prev) => !prev)
+
+  return (
+    <>
+      <FormProvider {...formMethods}>
+        <form onSubmit={formMethods.handleSubmit(onSubmit)} className={commonCss.form}>
+          <TxCard>
+            <Typography variant="h5">Trusted guardian</Typography>
+
+            <Typography variant="body2">
+              Choose a guardian, such as a hardware wallet or family member&apos;s wallet, that can initiate the
+              recovery process in the future.
+            </Typography>
+
+            <AddressBookInput
+              label="Guardian address"
+              name={`${EnableRecoveryFlowFields.guardians}.0`}
+              required
+              fullWidth
+            />
+
+            <Typography variant="h5">Recovery delay</Typography>
+
+            <Typography variant="body2">
```

Comment:
The spaces between the title and text looks a bit too big compared to the [designs](https://www.figma.com/file/pO43pjVwR34Uy1ihMsY3OM/Generic-recovery?node-id=677%3A30972&mode=dev). It also affects other titles/texts but I won't comment on all of them.

Reply:
Fixed in 2a78391b6029fb5b7ddaa5c84656125e7f7f9333.

Reply:
The title `Receive email updates` still has too large spacings. Rest looks good 👍

Reply:
Fixed in 7244e2e4432f56d3ce0355d9e2ebdfe383a652a5.